Java/core/src/main/java/com/amazon/randomcutforest/store/PointStore.java

```diff
@@ -710,6 +689,11 @@ public T capacity(int capacity) {
             return (T) this;
         }
 
+        public T initialSize(int initialPointStoreSize) {
```

Comment:
In other places we use `size` to mean "the number of things in this container". If that's what this value mean, I would just call it "size".

Reply:
its initialSize ... if dynamicresizing is allowed, then we can start from a small size an not allocate the worst case storage.